### PR TITLE
[feat] P0 features for New Tab

### PR DESCRIPTION
**Pull request by saurabhan**
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,75 @@
...
+    const response = await fetch(url);
+    const data = await response.json();
+    setQuote(data.body);
+  };
+
+  const getWeather = async (url) => {
+    const response = await fetch(url);
+    const data = await response.json();
+    console.log(data)
+    setWeather(data);
```
Comment (amartya-rs) [inline src/App.js:24]: remove console log

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,75 @@
...
+import Onboarding from "./component/Onboarding";
+import Content from "./component/Content";
 
 function App() {
+  const [quote, setQuote] = useState("");
+  const [weather, setWeather] = useState();
+  const [location, setLocation] = useState("delhi");
+  const user = useState(localStorage.getItem("user"));
+  console.log(user)
+  const api_key = process.env.REACT_APP_WEATHER_API;
```
Comment (amartya-rs) [inline src/App.js:11]: remove console log

**Review by amartya-rs — APPROVED** [review]
Code looks fine to me. Good to merge.

**Review by archit7448 — APPROVED** [review]
code looks good to merge.

**Pull request merged**